### DISCUSSION TITLE
Dark Theme

### DISCUSSION THREAD

**Discussion by bloor** [melt-ui/melt-ui#474] [Ideas]
Pleaaaaaaaaaaaaaaaaaaaaaaaase. Out of the box dark theme. 
Don't be like other UI component dealers in my area.

PS: The component list is comprehensive and has close attention to details. My favorite has to be the dropdown and tags-input👍

**Comment by TGlide** [comment]
> Pleaaaaaaaaaaaaaaaaaaaaaaaase. Out of the box dark theme. Don't be like other UI component dealers in my area.
> 
> PS: The component list is comprehensive and has close attention to details. My favorite has to be the dropdown and tags-input👍

Melt doesn't have neither light or dark themes, it's an unstyled library 😅

We do have examples, but that's it 🙂
